tests/flow/test_ts_range.py

```diff
@@ -7,6 +7,7 @@
     _insert_agg_data
 from includes import *
 from ctypes import *
+from utils import timeit
```

Comment:
@OfirMos you are not using it.

Reply:
@danni-m  I was using it to measure how long the test will take, I think it's good to have it for when I need to measure time of tests